app/src/main/java/pl/pola_app/helpers/Utils.java

```diff
@@ -51,4 +50,10 @@ public static int dpToPx(int dp) {
     public static int pxToDp(int px) {
         return (int) (px / Resources.getSystem().getDisplayMetrics().density);
     }
+
+    public static void logException(Throwable throwable) {
+        if (!BuildConfig.USE_CRASHLYTICS) {
+            Crashlytics.logException(throwable);
```

Comment:
We use crashlytics logging through Timber and its CrashReportingTree (check PolaApplication.java line 55). I guess it makes sense to add logException there and stick to Timber.log